cl/assets/templates/base.html

```diff
@@ -165,13 +165,10 @@ <h1>You did not supply the "private" variable to your template.
             <span class="icon-bar"></span>
           </button>
           <a class="navbar-brand hidden-xs" href="/">
-            <img src="{% static "svg/logo-full.svg" %}"
-                 alt="CourtListener"
-                 height="40"
-                 tabindex="1"/>
+            {% include "includes/cl_logo_svg.html" %}
```

Comment:
I think you can name the file whatever you want and do:

```suggestion
            {% include "includes/cl_logo.svg" %}
```

Pretty sure there's no requirement to include only `html` files.

Reply:
Yeah it works well using the SVG extension, changed it.

Reply:
i was poking around and found this very useful 🕵️

Reply:
We use this approach a lot on bots.law.